src/policy/feerate.cpp

```diff
@@ -7,29 +7,30 @@
 
 #include <tinyformat.h>
 
-CFeeRate::CFeeRate(const CAmount& nFeePaid, size_t nBytes_)
+CFeeRate::CFeeRate(const CAmount& nFeePaid, uint32_t num_bytes)
 {
-    assert(nBytes_ <= uint64_t(std::numeric_limits<int64_t>::max()));
-    int64_t nSize = int64_t(nBytes_);
+    int64_t nSize = int64_t{num_bytes};
```

Comment:
use a named cast? https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-casts-named

Reply:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es23-prefer-the--initializer-syntax